ynput/ayon-applications
# Add ShapeFX Loki application

## Changelog Description

Add [ShapeFX Loki](https://shapefx.app/) support.

## Additional info

Requires Loki in [`ayon-loki` integration](https://github.com/BigRoy/ayon-loki).

## Testing notes:

1. Install ShapeFX Loki
2. Run through AYON
3. Basics should work.
